<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>658</width>
                <height>504</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="styleSheet">
            <string notr="true"/>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QGroupBox" name="groupBox">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>80</y>
                        <width>621</width>
                        <height>381</height>
                    </rect>
                </property>
                <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                </property>
                <property name="title">
                    <string>Video</string>
                </property>
                <widget class="QWidget" name="ImgWidget" native="true">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>20</y>
                            <width>601</width>
                            <height>351</height>
                        </rect>
                    </property>
                </widget>
            </widget>
            <widget class="QPushButton" name="startButton">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>170</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Start video</string>
                </property>
            </widget>
            <widget class="QPushButton" name="stopButton">
                <property name="geometry">
                    <rect>
                        <x>230</x>
                        <y>20</y>
                        <width>170</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Stop video</string>
                </property>
            </widget>
            <widget class="QPushButton" name="exportButton">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>20</y>
                        <width>170</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Export report</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>658</width>
                    <height>25</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
